CMake/CheckAndAddFlag.cmake

```diff
@@ -3,13 +3,13 @@ include(CheckCXXCompilerFlag)
 
 # check_add_add_flag(<variable> <flag> [DEBUG_ONLY | RELEASE_ONLY])
 #
-# Add a C or C++ compilation flag to the current scope
+# Add C or C++ compilation flags to the current scope
```

Comment:
Please revert this change. As the name and signature indicate, `check_and_add_flag` is used to add a flag, not several flags.

Reply:
Done. The multiple examples made it seem like multiple flags could be added.